bucket/unikey.json

```diff
@@ -0,0 +1,34 @@
+{
+    "version": "43RC5-200929",
+    "description": "UniKey is the most popular Vietnamese keyboard for Windows. The core UniKey Vietnamese Input Method is also the engine for many Vietnamese keyboards in Linux, macOS and iOS. UniKey is free of charge. Source code for the UniKey Vietnamese Input Method is distributed under the GNU General Public License.",
+    "homepage": "https://www.unikey.org",
+    "license": {
```

Comment:
use `"license": "GPL-3.0-or-later"` instead.
Identifiers can be found on https://spdx.org/licenses/